pepper-apis/.gitignore

```diff
@@ -32,4 +32,5 @@ appengine/deploy/*.conf
 .java-version
 
 appengine
-.idea
\ No newline at end of file
+.idea
+/config/dev-gcloud-creds.json
```

Comment:
What is this?

Reply:
Added configuration file with Google cloud secrets to git ignore to avoid accident commit